tests/model_fields_/test_embedded_model.py

```diff
@@ -123,3 +129,62 @@ class MyModel(models.Model):
         self.assertEqual(
             msg, "Embedded models cannot have relational fields (Target.key is a ForeignKey)."
         )
+
+
+class SubqueryExistsTest(TestCase):
+    def setUp(self):
+        # Create test data
+        address1 = Address.objects.create(city="New York", state="NY", zip_code=10001)
+        address2 = Address.objects.create(city="Boston", state="MA", zip_code=20002)
+        author1 = Author.objects.create(name="Alice", age=30, address=address1)
+        author2 = Author.objects.create(name="Bob", age=40, address=address2)
+        book1 = Book.objects.create(name="Book A", author=author1)
```

Comment:
It would be nice to title it "Book 1" rather than switching between letters and numbers.